documentation/modules/managing/con-cluster-loss-recovery.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-loss_{context}"]
+= Recovery from loss of a Kafka cluster
```

Comment:
I think in the original document I meant this more as Kubernetes cluster loss, not Kafka loss.

Reply:
Changed to _Recovery from loss of a Kubernetes cluster_